core-foundation/src/propertylist.rs

```diff
@@ -86,8 +86,8 @@ pub trait CFPropertyListSubClass: TCFType {
 
 impl CFPropertyListSubClass for ::data::CFData {}
 impl CFPropertyListSubClass for ::string::CFString {}
-impl CFPropertyListSubClass for ::array::CFArray {}
-impl CFPropertyListSubClass for ::dictionary::CFDictionary {}
+impl<T> CFPropertyListSubClass for ::array::CFArray<T> {}
+impl<K, V> CFPropertyListSubClass for ::dictionary::CFDictionary<K, V> {}
```

Comment:
How stupid of me.. This of course applies to for example `impl<T> CFPropertyListSubClass for ::array::CFArray<T> {}` as well. That would allow downcasting to a typed array directly, which would not be safe. I removed the generics on these `CFPropertyListSubClass` implementations again..